helix-core/src/main/java/org/apache/helix/cloud/CloudProvider.java

```diff
@@ -0,0 +1,25 @@
+package org.apache.helix.cloud;
```

Comment:
Let's create a constant folder and move this to the /helix/could/constants, since we will have other constants there.

Reply:
Good suggestion. Done.